Classes/Wwwision/Neos/MailChimp/Domain/Service/MailChimpService.php

```diff
@@ -76,6 +83,44 @@ public function getListById($listId)
 
     /**
      * @param string $listId
+     * @param string $interestCategoryId
+     * @return array
+     */
+    public function getCategoryByListIdAndInterestCategoryId($listId, $interestCategoryId)
+    {
+        $cacheKey = "MailChimp_List_" . $listId ."_Category_" . $interestCategoryId;
+        if ($this->cache->has($cacheKey)) {
+            return $this->cache->get($cacheKey);
+        }
+
+        $categoryResult = $this->get("lists/$listId/interest-categories/$interestCategoryId");
+
+        $this->cache->set($cacheKey, $categoryResult, [], 60 * 60 * 1); // 1 hour caching
```

Comment:
1 hour. Seems a bit arbitrary. Any specific reason for why you chose this duration?